Kitodo/src/test/java/org/kitodo/production/exporter/download/ExportMetsIT.java

```diff
@@ -11,6 +11,9 @@
 
 package org.kitodo.production.exporter.download;
 
+import static org.kitodo.utils.ProcessTestUtils.META_XML;
+import static org.kitodo.utils.ProcessTestUtils.METADATA_DIR;
+
```

Comment:
Codacy issue
"Kitodo/src/test/java/org/kitodo/production/exporter/download/ExportMetsIT.java#L15
Wrong order for 'org.kitodo.utils.ProcessTestUtils.METADATA_DIR' import."